guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -64,6 +64,11 @@ ifdef::katello,orcharhino,satellite[]
 include::topics/upgrading_content_hosts.adoc[leveloffset=+2]
 endif::[]
 
+ifdef::foreman-el,katello,satellite,orcharhino[]
+// Upgrading the External database
```

Comment:
Does this comment really add anything? I know we have it all over the place, but it's essentially repeating the self-explaining filename.

Reply:
@ekohl I have often wondered the same thing.